cfy_manager/utils/service.py

```diff
@@ -147,12 +173,21 @@ def _get_full_service_name(service_name, append_prefix):
 
 
 class Supervisord(object):
-    def supervisorctl(self, action, service='', ignore_failure=False):
+    def supervisorctl(
+            self,
+            action,
+            service='',
+            ignore_failure=False,
+            options=None
+    ):
+        options = options or []
         cmd = [
             'supervisorctl', '-c', '/etc/supervisord.conf', action
         ]
         if service:
             cmd += [service]
+        if options:
```

Comment:
This could be rewritten as cmd.extend(options)
(no need for the conditional)

Reply:
Done